corehq/apps/users/models.py

```diff
@@ -382,6 +382,7 @@ class DomainMembership(Membership):
     override_global_tz = BooleanProperty(default=False)
     role_id = StringProperty()
     location_id = StringProperty()
+    assigned_location_ids = StringListProperty()
```

Comment:
@sheelio Shouldn't web-users also have the multiple-locations field (they already have the location_id field)?